### ISSUE TITLE
Form attribute - adding email example

### ISSUE THREAD

**Issue by lukefretwell** [proudcity/wp-proudcity#1900]
> Name, Email, and Phone form fields are missing autocomplete attributes that make filling out the forms easier. Recommend adding autocomplete="name", autocomplete="given-name", autocomplete="family-name", autocomplete="email", and autocomplete="tel". See Mozilla's help page on HTML autocomplete.

**Comment by jlyon** [comment]
Verified that this is not an available option in Gravity Forms. There are lots of available options to add to fields, but the `autocomplete` attribute is not one of the options that is configurable via the UI.

**Comment by jlyon** [comment]
This is a likely a cannot fix item @kevindherman
